### DISCUSSION TITLE
Creating a new remote repo from command line

### DISCUSSION THREAD

**Discussion by happydogcode** [community/community#21675] [New to GitHub]
<p>Is there any git command to create a new <em>remote</em> repo and push my commit to GitHub from there? I know I can just create a new repo on <a href="http://github.com" rel="nofollow noopener">github.com</a>, but if there is a way to achieve this from the CLI that would be great.</p>

**Comment by winstliu** [comment, marked as answer]
<p>Unfortunately, there is no native git command to create a new GitHub repo.  However, not all is lost.  GitHub maintains a command-line wrapper for git called hub which conveniently allows you to create a new GitHub repository from the CLI:</p>
<pre><code>git **create -d "\&lt;commit name\&gt;"**
</code></pre>
<p>This will create a new repository called whatever the current directory is named.  If you’re interested, you can find out more about hub <a href="https://hub.github.com/" rel="nofollow noopener">here</a>.</p>
<p>I hope that helped!</p>

**Comment by Jamariomiller27** [comment]
<p>Thanks for the statement you gave to me i really appreciate that i can learn alot from this business</p>

**Comment by ghost** [comment]
<p>It returns:</p>
<p>git: ‘create’ is not a git command. See ‘git --help’.</p>
<p>The most similar command is<br>
reset</p>

**Comment by carlinmack** [comment]
<p>You need to install hub at the <a href="https://hub.github.com" rel="nofollow noopener">link posted above</a> </p>

**Comment by ArnarFreyrKristinsson** [comment]
<p>Its hub instead of git: </p>
<pre><code>&lt;font color="#FF0000"&gt;hub&lt;/font&gt; **create -d "\&lt;commit name\&gt;"**
</code></pre>
